compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -67,13 +67,15 @@ class SystemFont(
  */
 class LoadedFont internal constructor(
     override val identity: String,
-    val data: ByteArray,
+    internal val getData: () -> ByteArray,
     override val weight: FontWeight,
     override val style: FontStyle
 ) : PlatformFont() {
     @ExperimentalTextApi
     override val loadingStrategy: FontLoadingStrategy = FontLoadingStrategy.Blocking
 
+    val data: ByteArray by lazy { getData() }
```

Comment:
Disagree with adding `lazy` addition. The whole purpose of this PR was to avoid storing this byte array in memory

Reply:
Ok, if @terrakok not have a strong opinion let's revert it to simple function call